Calorie progress bar initial commit

mmoren01 opened the pull request:
Adding the calorie progress bar as is for demo purposes. To Do's: Hook it up to live data

Commit:
Created component jsx and imported apple.svg into public for progress bar use
Rebasing to latest testing branch changes

Commit:
Added an apple avg file
Added package @svgr/webpack to allow SVG file support
Added an config.next.js file to modify our webpack to use SVGR package

Commit:
Rebased branch to latest testing branch

Commit:
Imported required bootstrap css in _app.js for global use
Created a calorie tracker specific css file in styles
Imported calorie track specific css file in _app.js as required by Next.js

Added bootstrap as a require package for react bootstrap

Commit:
Completed MVP graphic for calorie tracker
Will work on tying in functionality to the progress bar next

Commit:
Cleaned up some ESlint issues

Commit:
Merge branch 'testing' of https://github.com/Nvision-Nutrition/nvision into calorieProgressBar

Commit:
Added a test button until hooked up to database

mmoren01 committed:
Delete .vscode directory

Deleted a vscode settings file that slipped in

emmaknor commented on components/CalorieTracker.jsx:
```diff
@@ -0,0 +1,43 @@
+import ProgressBar from 'react-bootstrap/ProgressBar';
+
+const CalorieTracker = () => {
+  const [progress, setProgress] = useState(60);
```
why is this initial state set to 60?

mmoren01 replied:
For testing. Will be set to zero once we start pulling data.

emmaknor commented on components/dailyTracker.jsx:
```diff
@@ -1,11 +1,13 @@
     <>
       <div className={`${styles.card} daily-tracker`}>
         <p className={`${styles.description} subtitle`}>{`Daily Tracker`}</p>
+        <CalorieTracker />
```
we may have some merge conflicts here, but we can sort it out!

mmoren01 replied:
Will address this once the previous PRs are approved

benkennedy98 reviewed (commented):
Looks good!

emmaknor commented on pages/_app.js:
```diff
@@ -1,6 +1,8 @@
 /* eslint-disable require-jsdoc */
 import '../styles/globals.css';
+import '../styles/calorie-progress-bar.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
another potential merge conflict

mmoren01 replied:
Same as last comment. Will address when previous PR's are merged

emmaknor commented on public/appleStencil.svg:
```diff
@@ -0,0 +1 @@
+<svg id="svg" version="1.1" width="300" height="259" viewBox="0 0 300 259" xmlns="http://www.w3.org/2000/svg"><path d="M0.000 129.500 L 0.000 259.000 150.000 259.000 L 300.000 259.000 300.000 129.500 L 300.000 0.000 150.000 0.000 L 0.000 0.000 0.000 129.500 M217.580 4.511 C 224.138 8.327,224.555 9.339,220.765 12.230 C 218.969 13.599,214.575 16.190,211.000 17.987 C 199.768 23.634,194.146 27.935,18 ...
```
is this all one line? looks like some lint to me

mmoren01 replied:
It's an image file from a compiler. Should be okay to disregard. Please let me know if this needs to be addressed

nohren reviewed (commented):
Nice work!

Commit:
Merge branch 'testing' of https://github.com/Nvision-Nutrition/nvision into calorieProgressBar

Commit:
Merge branch 'testing' of https://github.com/Nvision-Nutrition/nvision into calorieProgressBar

Commit:
Resolved conflicts with testing branch
Removed packages that were not longer needed

Commit:
Debugged push error

Merge branch 'calorieProgressBar' of https://github.com/Nvision-Nutrition/nvision into calorieProgressBar
